src/ensembl/src/content/app/help/hooks/useHelpAppAnalytics.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import analyticsTracking from 'src/services/analytics-service';
+
+const useHelpAppAnalytics = () => {
+  const trackTopLevelMenu = (navLabel: string) => {
+    analyticsTracking.trackEvent({
+      category: 'help_mega_nav',
+      label: navLabel,
+      action: 'opened'
+    });
+  };
+
+  const trackArticleLinkClicked = (linkLabel: string, type: string) => {
```

Comment:
- `clicked` -> `click` (see useEntityViewerAnalytics for comparison)
- maybe `trackMegaNavItemClick`? The reasons being that
  - this would indicate that the clicked thingy is specifically in the meganav (as opposed to any other place)
  - the clicked thingy can be a link to an article or a link to a video